content/docs/communities/bridge-discord/_index.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Bridging my Discord community"
+weight = 300
+[extra]
+emoji = "🌉"
+tile = "I want to bridge my Discord community to Matrix"
+last_updated = "2022-10-18T16:00:00Z"
++++
+
+The easiest way to bridge your Matrix community with you Discord one is to rely
+on the free Discord bridge [provided by t2bot.io](https://t2bot.io/discord/).
+
+It contains all the instructions about how to bridge your Discord channels with
+your Matrix rooms. Please refer to their documentation directly.
+
+Here you can see a bridged room from the Matrix and the Discord perspective.
+
+![](./from-matrix.png)
+![](./from-discord.png)
+
+If you're interested in self-hosting your bridge or discovering all the Discord
+bridge implementations, please head to the [Ecosystem > Bridges](/ecosystem/bridges)
```

Comment:
I really don't like angle brackets in text. `›` should look better (and this one is available on any regular system).

Reply:
I didn't use those for now because we may want to discuss it further. My major concern is that this character is not easily available on keyboards, I'd rather find a solution for which we have better consistency guarantees.

Reply:
It's available as `&rsaquo;`, does that help? 😅